modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -159,6 +163,16 @@ The API console request panel component shows a panel where users can test an AP
 * *Headline Font Family (Optional)*: Set the font family of the headline.
 * *Error Color (Optional)*: Set the color of error messages.
 
+== API Details Button
+
+The API details button enables users to access details about an API.
+
+* *API*: Choose the API to show.
+* *Button Label*: Set the button label. Default is `API Details`.
+* *Button Align*: Set button alignment to left, right, or center.
+* *API Name*: Choose "Append", "Prepend", or "None" to add the API name to the button label.
+* *API Version*: Choose "Append", "Prepend", or "None" to add the API version to the button label.
```

Comment:
A few minor suggestions:
*API*: Choose the API to show. (Same comment as above - 'show' vs 'display')
* *Button Label*: Specifies the button label. Default is `API Details`.
* *Button Align*: Set button alignment to left, right, or center. (Of what? What will this be next to in the UI?)
* *API Name*: Add the API name to *Button Label*. Values include "Append", "Prepend", or "None". What is the default here?
* *API Version*: Add the API version to the *Button Label*. Values include "Append", "Prepend", or "None". What is the default here?

Also, since the values are UI elements, should they be bolded?

Reply:
Good catch, those should be bold. Updated with these changes.